src/components/accordion/Accordion.stories.js

```diff
@@ -12,26 +12,47 @@ export default {
   title: 'Work in Progress/Accordion',
   component: `kd-accordion`,
   argTypes: {
+    filledHeaders: {
+      control: { type: 'boolean' },
+    },
+    compact: {
+      control: { type: 'boolean' },
+    },
     showNumbers: {
       control: { type: 'boolean' },
     },
     startNumber: {
       control: { type: 'number' },
     },
+    expandLabel: {
+      control: { type: 'text' },
+    },
+    collapseLabel: {
+      control: { type: 'text' },
+    },
```

Comment:
You don't need to add argTypes here. Controls are automatically generated based on your JSDoc code comments when running the `analyze` command.